bandwidth-file-spec.txt

```diff
@@ -57,15 +59,15 @@
 
   All Tor versions can consume format version 1.0.0.
   All Tor versions can consume format version 1.1.0 and later,
-  but they warn on additional header Lines.
-  [TODO: this might be fixed, and if it is fixed should be said which
-  version of Tor]
+  but Tor versions earlier than 0.3.5.1-alpha warn if the header
+  contains any KeyValue lines after the Timestamp.
 
 2. Format details
 
   The Bandwidth List MUST contain the following sections:
-  - Header List (exactly once)
-  - Relays' Bandwidth List (zero or more times)
+  - Header List (exactly once), which is an ordered list of
```

Comment:
You're right, only the first few fields are ordered.
Timestamp has to be first because that's what tor expects.
Version needs to be next, because parsers need to know the version to parse the rest of the file.

Reply:
Fixed in 9b206ed.